probe-rs/src/probe/daplink/tools.rs

```diff
@@ -170,11 +192,7 @@ pub fn open_device_from_selector(
             // multiple open handles are not allowed on Windows.
             drop(handle);
 
-            if d_desc.vendor_id() == selector.vendor_id
-                && d_desc.product_id() == selector.product_id
-                && sn_str == selector.serial_number
-                && get_daplink_info(&device).is_some()
```

Comment:
Is it intentional to remove `get_daplink_info().is_some()` check?

Reply:
Nope, I readded it :)